src/applications/vaos/new-appointment/components/DateTimeRequestPage/CommunityCare.jsx

```diff
@@ -122,15 +131,17 @@ export default function CCRequest() {
 
           return dispatch(routeToPreviousAppointmentPage(history, pageKey));
         }}
-        onSubmit={() =>
+        onSubmit={() => {
+          // Increment the focus trigger to force re-focusing the validation message
+          setFocusTrigger(prev => prev + 1);
```

Comment:
This approach is inspired by https://github.com/department-of-veterans-affairs/vets-website/pull/37119. From testing, it seems this is only necessary for highly customized pages like date time pages that uses the `CalendarWidget`. Regular pages using the normal form validation do not need this forced focus trigger.